hw1/02-palindrome.js

```diff
@@ -1,3 +1,25 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput() {
+  const value = parseFloat(elem.value);
+  const resultDiv = document.querySelector("#result");
+
+  if (value >= 0) {
+    const numString = value.toString();
+    const reverseString = numString.split("").reverse().join("");
+
+    if (numString == reverseString) {
+      resultDiv.textContent = "Yes. This is a palindrome!";
+      document.getElementById("result").style.color = "green";
```

Comment:
explicit setting of style color can be avoided by just using the bootstrap .

Reply:
Sounds good, I'll have to look at how to do that as I previously wasn't able to find a font color option within bootstrap.